### ISSUE TITLE
Variable "symbol" does not exist

### ISSUE THREAD

**Issue by jaimescript** [symfony/symfony#42101]
Symfony version(s) affected: 4.4.26

Description

Variable "symbol" does not exist. in  vendor/symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig (line 201)

How to reproduce
Create a simple form using the FormBuilder

Possible Solution

From : {%- if symbol is not empty -%}
To: {%- if symbol is defined and symbol is not empty -%}

I´ve read about a similar issue, and it says about symfony/form and version of symfony/twig-bridge https://github.com/symfony/symfony/issues/27489#issuecomment-394352485

Im not sure if it is a bug or a version compatibility version of packages related to Twig

**Comment by xabbuh** [comment]
Do you have a custom form type that returns `percent` as the block prefix?

**Comment by xabbuh** [comment]
I am going to close here for now due to the lack of feedback. Please let us know when you have more information and we can consider to reopen.

**Comment by loicpennamen** [comment]
Solution in my case: I had the same issue, somehow I figured my form type's name was the problem. On Symfony 6.3 with all updated dependencies, I created an `App\Form\Type\PercentType` class (basically a glorified `IntegerType`).

The bug happens.

When I change the name of my class to `App\Form\Type\PctType` the error goes away. It is quite a funny bug, unfortunately no time right now to dig deeper into a fix proposal for Symfony.
